lib/plugins/helper/meta_generator.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 
-function metaGeneratorHelper() {
-  return `<meta name="generator" content="Hexo ${this.env.version}">`;
-}
+let metaGeneratorTag = '';
 
-module.exports = metaGeneratorHelper;
+module.exports = ctx => {
+  return () => {
+    if (!ctx.extend.helper.getProp('meta_generator')) ctx.extend.helper.setProp('meta_generator', true);
```

Comment:
Suggest `use_meta_generator_helper` as the property name.
If `meta_generator` is used, we don't know what this means.

Reply:
Change to `meta_generator_helper_used`